packages/react/src/_collapsing-side-bar/CollapsingSideBar.tsx

```diff
@@ -1,58 +1,58 @@
 import { PropsWithChildren, ReactNode, useRef } from 'react';
 import { useSpring, animated } from '@react-spring/web';
 import {
-	mapResponsiveProp,
-	mq,
+	packs,
 	tokens,
 	usePrefersReducedMotion,
 	useToggleState,
-	useWindowSize,
 } from '@ag.ds-next/react/core';
-import { backgroundColorMap, Box } from '@ag.ds-next/react/box';
+import { Box, type BoxProps } from '@ag.ds-next/react/box';
 import { BaseButton } from '@ag.ds-next/react/button';
 import { Flex } from '@ag.ds-next/react/flex';
-import { Stack } from '@ag.ds-next/react/stack';
 import { ChevronDownIcon } from '@ag.ds-next/react/icon';
+import { Stack } from '@ag.ds-next/react/stack';
+import { Text } from '@ag.ds-next/react/text';
 import {
-	hoverColorMap,
 	CollapsingSideBarBackground,
-	collapsingSideBarLocalPalette,
-	collapsingSideBarLocalPaletteVars,
+	collapsingSideBarHoverProp,
+	collapsingSideBarHoverVar,
+	hoverColorMap,
 	useCollapsingSideBarIds,
 } from './utils';
 
 export type CollapsingSideBarProps = PropsWithChildren<{
 	/** The HTML element to render the CollapsingSideBar as. */
 	as?: CollapsingSideBarContainerElementType;
-	/** The aria-label for the CollapsingSideBar. */
-	'aria-label'?: string;
 	/** If CollapsingSideBar is placed on 'bodyAlt' background, please set this to 'bodyAlt'. */
 	background?: CollapsingSideBarBackground;
-	/** Used as the title of the expand/collapse trigger on smaller screen sizes. */
-	collapseButtonLabel: string;
-	/** The title of the CollapsingSideBar. Place a CollapsingSideBarTitle here unless you implement something custom. */
-	title?: ReactNode;
-	/** The sub title of the CollapsingSideBar. */
+	/** A custom element to use instead of title and subTitle. It should render as an h2. */
```

Comment:
Does that mean you have to pass it a `h2`?

Reply:
Is it worth warning if it isn't a `h2`? Or adjusting the JSDoc to call out accessibility?

```suggestion
	/** A custom element to use instead of title and subTitle. To make this accessible, it should render as an h2. */
```

Reply:
I'm not sure how I could warn since anything could be passed? I'll improve the JSDoc though